applications/teletype/src/teletype_templates.erl

```diff
@@ -163,20 +163,38 @@ templates_source(_TemplateId, 'undefined', _ResellerId) ->
     'undefined';
 templates_source(TemplateId, AccountId, AccountId) ->
     case fetch_notification(TemplateId, AccountId) of
-        {'ok', Template} -> kz_doc:account_id(Template);
+        {'ok', Template} ->
+            templates_source_has_attachments(TemplateId, AccountId, AccountId, Template);
         {'error', 'not_found'} -> ?KZ_CONFIG_DB;
         {'error', _E} -> 'undefined'
     end;
 templates_source(TemplateId, AccountId, ResellerId) ->
     case fetch_notification(TemplateId, AccountId) of
         {'error', 'not_found'} ->
-            lager:debug("failed to find template ~s in account ~s", [TemplateId, AccountId]),
-            ParentId = teletype_util:get_parent_account_id(AccountId),
-            templates_source(TemplateId, ParentId, ResellerId);
-        {'ok', Template} -> kz_doc:account_id(Template);
+            parent_templates_source(TemplateId, AccountId, ResellerId);
+        {'ok', Template} ->
+            templates_source_has_attachments(TemplateId, AccountId, ResellerId, Template);
         {'error', _E} -> 'undefined'
     end.
 
+-spec templates_source_has_attachments(ne_binary()
+                                      ,ne_binary()
+                                      ,ne_binary()
+                                      ,kz_json:object()) -> api_binary().
```

Comment:
@danielfinke can you please make it one line? I don't thing we have a specific code style for line length (code or spec).
If we want to break long line, we usually will do it for just the code not specs, for long specs we usually put the return type into new line. 
For this spec specifically there's no need to break it into multi line.